UnitTesting/DependencyInjectionFixture.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DataAccess;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Services.Interfaces;
+using Services;
+using Domain;
+
+namespace UnitTesting
+{
+    public class DependencyInjectionFixture
+    {
+        public ServiceProvider ServiceProvider { get; private set; }
+        private string connectionString = "Server=localhost;Database=DotNetCourseDatabase;TrustServerCertificate=true;Trusted_Connection=true;";
```

Comment:
It would be a good idea to use a different db for testing than the one you use for running your API